tools/wptrunner/wptrunner/stability.py

```diff
@@ -278,7 +279,7 @@ def get_steps(logger, repeat_loop, repeat_restart, kwargs_extras):
     for kwargs_extra in kwargs_extras:
         if kwargs_extra:
             flags_string = " with flags %s" % " ".join(
-                "%s=%s" % item for item in kwargs_extra.iteritems())
+                "%s=%s" % item for item in iteritems(kwargs_extra))
```

Comment:
Note that this issue is already addressed in #22538. I'm just adding it here to make TC happy.